proposals/low-level-struct-improvements.md

```diff
@@ -421,10 +423,12 @@ The section on `ref` field and `scoped` is long so wanted to close with a brief
 
 Detailed Notes:
 - A `ref` field can only be declared inside of a `ref struct` 
-- A `ref` field cannot be declared `static`
+- A `ref` field cannot be declared `static`, `volatile` or `const`
 - The reference assembly generation process must preserve the presence of a `ref` field inside a `ref struct` 
 - A `readonly ref struct` must declare its `ref` fields as `readonly ref`
-- The span safety rules for method invocation, fields and assignment must be updated as outlined in this document.
+- The span safety rules
+    - Will be updated as outlined in this document.
+    - Will take effect whenever it's determined the core library supports `ref` fields
```

Comment:
Consider being more specific here. Are we talking about compile-time feature detection similar to what we do for DIMs?

Reply:
Updated this a bit